operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -485,6 +485,31 @@ rbac:
     runAsUser: 10001
     seccompProfile:
       type: RuntimeDefault
+  
+  # -- PSP configuration
+  podSecurityPolicy:
+    seccompProfile: runtime/default
+    privileged: false
+    allowPrivilegeEscalation: false
+    hostNetwork: false
+    hostIPC: false
+    hostPID: false
+    readOnlyRootFilesystem: false
```

Comment:
the previous default used to be true and mimir shouldn't need root FS support. Can you change this?

Reply:
Done